test/test-pages/medium-3/expected-metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-  "title": "Samantha and The Great Big Lie",
+  "title": "Samantha and The Great Big Lie - John C. Welch - Medium",
   "byline": "John C. Welch",
   "dir": null,
-  "excerpt": "How to get shanked doing what people say they want",
-  "readerable": true,
-  "siteName": "Medium"
+  "excerpt": "(EDIT: removed the link to Samantha’s post, because the arments and the grubers and the rest of The Deck Clique got what they wanted: a non-proper person driven off the internet lightly capped with a…",
```

Comment:
Hm, why did the excerpt change? It looks like the metadata in the source markup is still the same...

Reply:
I'm not entirely sure, but the original testcase source wasn't correct - it had a copy of the document, and then part of a Firefox crash report, and then another copy of the document. I think having two copies may have just confused JSDOM into doing something different.